packages/utils/typescript/lib/__tests__/generators/schemas/attributes.test.js

```diff
@@ -438,6 +438,61 @@ describe('Attributes', () => {
         });
       });
 
+      describe('Custom field', () => {
+        test('No custom field', () => {
+          const attribute = {};
+          const modifiers = getAttributeModifiers(attribute);
+
+          expect(modifiers).toHaveLength(0);
+        });
+
+        test('Basic custom field', () => {
+          const attribute = {
+            type: 'customField',
```

Comment:
Can you have `type: 'customField'` once the application is loaded? I thought it was mapped to the referenced type?